File: app-ios/ios/controller/AboutViewController.m
//
//  SecondViewController.m
//  ios
//
//  Created by Peter Zhan on 2017/10/15.
//  Copyright © 2017年 com.enuocms. All rights reserved.
//

#import "AboutViewController.h"

@interface AboutViewController ()
- (IBAction)btnLogoutClick:(id)sender;

@end

@implementation AboutViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)btnLogoutClick:(id)sender {
    [UserInfo setCurrentUserID:nil];
    [ShareAppDelegate fnShowLoginViewController];
}
@end

File: app-swift/ios/ios/controller/AboutViewController.swift
//
//  AboutViewController.swift
//  ios
//
//  Created by Peter Zhan on 2017/10/18.
//  Copyright © 2017年 com.enuocms. All rights reserved.
//

import UIKit

class AboutViewController : UIViewController{
    @IBAction func btnLogoutClick(_ sender: Any) {
        let uerinfo = UserInfo.init()
        uerinfo.saveInfo()
        Common.ShareAppDelegate().fnShowLoginViewController()
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
}
